config/azure/setBuildVars.py

```diff
@@ -52,32 +53,73 @@
 # trigger. These variables can be referenced in a pipeline yaml file downstream
 # of the step that runs this script.
 
+## SOURCE COMMIT HASH
```

Comment:
WHAT'S WITH THE SHOUTING?

Reply:
📣 Fixed in https://github.com/GafferHQ/gaffer/commit/fbd71b5757a4a2f9492fbcdb05261b6954c0e704